docs-gen/content/rule_set/data_entry/allowed.md

```diff
@@ -20,8 +20,6 @@ SteeringWheel.Position:
 
 ```
 
-If `allowed` is set, `min` or `max` cannot be defined.
```

Comment:
This is not enforced in vss-tools. In standard catalog as of today we only use `allowed` for strings, and there min/max does not make sense, but nothing that forbids it.

Reply:
In the same manner that "default" only applies to attributes, but nothing forbids assigning a default to other signals.